easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.3-fosscuda-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MesonNinja'
+
+name = 'libepoxy'
+version = '1.5.3'
+
+homepage = 'https://github.com/anholt/libepoxy'
+description = "Epoxy is a library for handling OpenGL function pointer management for you"
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/anholt/%(name)s/releases/download/%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['002958c5528321edd53440235d3c44e71b5b1e09b9177e8daf677450b6c4433d']
+
+builddependencies = [
+    ('Meson', '0.48.1', '-Python-3.6.6'),
+    ('Ninja', '1.8.2'),
+]
+
+dependencies = [
+    ('X11', '20180604'),
+    ('Mesa', '18.1.1'),
+]
+
+configopts = '-Degl=no --libdir %(installdir)s/lib '
+
+sanity_check_paths = {
+    'files': ['include/epoxy/%s' % x for x in ['common.h', 'gl_generated.h', 'gl.h', 'glx_generated.h', 'glx.h']],
+    'dirs': ['lib']
```

Comment:
@akesandgren We should check for `'lib/libepoxy.%s' % SHLIB_EXT`, since the header files are only being copied, not built?

Reply:
Done